src/components/ImportLogsDetail.vue

```diff
@@ -0,0 +1,281 @@
+                <ion-icon slot="start" :icon="codeWorkingOutline" />
+                {{ translate('Files processed') }}
+                <ion-label slot="end">{{ getProcessedFileCount() }}</ion-label>
+              </ion-item>
+              <ion-item lines="none">
+                <ion-icon slot="start" :icon="warningOutline" />
+                {{ translate('Files with errors') }}
+                <ion-label slot="end">{{ getErrorFileCount() }}</ion-label>
+              </ion-item>
+            </ion-list>
+          </section>
+        </div>
+        <div class="config ion-padding">
+          <ion-label lines="none">
+            <p class="overline">{{ currentJob.runtimeData?.configId }}</p>
+            <h1>{{ configDetails?.description }}</h1>
+          </ion-label>
+          <ion-list>
+            <ion-item>
+              <ion-icon slot="start" :icon="shareSocialOutline" />
+              {{ translate('Multithreading') }}
+              <ion-label slot="end">{{ currentJob.runtimeData?.groupBy ? currentJob.runtimeData?.groupBy : "-" }}</ion-label>
+            </ion-item>
+            <ion-item>
+              <ion-icon slot="start" :icon="globeOutline" />
+              {{ translate('SFTP') }}
+              <ion-label class="ion-text-wrap" slot="end">{{ configDetails?.importPath }}</ion-label>
+            </ion-item>
+            <ion-item>
+              <ion-icon slot="start" :icon="optionsOutline" />
+              {{ translate('Mode') }}
+              <ion-label slot="end">{{ configDetails?.executionModeId ? configDetails?.executionModeId : "-"}}</ion-label>
+            </ion-item>
+          </ion-list>
+        </div>
+      </div>
+
+      <div class="ion-padding">
+        <ion-chip v-for="(chip, index) in chips" :key="index" outline @click="updateLogs(chip.label)">
+          <ion-label>{{ chip.label }}</ion-label>
+          <ion-icon v-if="selectedChip === chip.label" :icon="checkmarkOutline" />
+        </ion-chip>
+      </div>
+      
+      <div class="empty-state" v-if="isLoading">
+        <ion-spinner name="crescent" />
+      </div>
+      <div v-else-if="filteredLogs?.length" >
+        <div class="list-item" v-for="(log, index) in filteredLogs" :key="index">
+          <ion-item lines="none">
+            <ion-icon slot="start" :icon="documentTextOutline" />
+            <ion-label>
+              <p class="overline">{{ log.logId }}</p>
+              {{ log.contentName }}
+              <p>{{ getDateTime(log.createdDate) }}</p>
+            </ion-label>
+          </ion-item>
+  
+          <ion-label>
+            {{ getDateTime(log.startDateTime) }}
+            <p>{{ translate('Started') }}</p>
+          </ion-label>
+  
+          <ion-label>
+            {{ getDateTime(log.finishDateTime) }}
+            <p>{{ translate('Finished') }}</p>
+          </ion-label>
+          
+          <ion-badge color="success" v-if="log.statusId === 'SERVICE_FINISHED'">{{ translate('Finished') }}</ion-badge>
```

Comment:
Display the status description here. You need to fetch the status data for statusTypeId='SERVICE_STATUS' and then create uiLables for all the statuses.